packages/xdl/src/project/Doctor.js

```diff
@@ -432,6 +436,77 @@ export async function validateWithNetworkAsync(projectRoot: string): Promise<num
   return validateAsync(projectRoot, true);
 }
 
+export async function hasWebSupportAsync(projectRoot, exp) {
+  let inputExp = exp;
+  if (!exp) {
+    inputExp = (await ProjectUtils.readConfigJsonAsync(projectRoot)).exp;
+  }
+  const { platforms } = inputExp;
+  if (!Array.isArray(platforms)) {
+    return false;
+  }
+
+  const isWebConfigured = platforms.includes('all') || platforms.includes('web');
```

Comment:
Is `all` actually valid value in the platforms array? Not sure if we want to add this to the schema.

Reply:
I saw it somewhere, if it's deprecated I can remove it.

Reply:
Only `android`, `ios` and `web` values are allowed: https://github.com/expo/universe/blob/94351697e532afedcd51b672f91b3e07716f9b25/server/www/xdl-schemas/UNVERSIONED-schema.json#L44

```suggestion
  const isWebConfigured = platforms.includes('web');
```